Library/Formula/lldpd.rb

```diff
@@ -20,6 +20,8 @@ class Lldpd < Formula
   depends_on "jansson"  if build.with? "json"
 
   def install
+    (var/"run").mkpath
```

Comment:
Instead of putting this in `install`, can you do it in `postinstall`? That way it should also work for binary packages.